ultra/ultra/scenarios/common/distributions.py

```diff
@@ -284,8 +284,8 @@
                 "distribution": behavior_distribution,
                 "start_end_on_different_lanes_probability": 0.0,
                 "begin_time_init": {
-                    "func": basic_begin_time_init_func,
-                    "params": {"probability": prob_heavy},
+                    "func": poisson_init,
+                    "params": {},
```

Comment:
Would you say the main fix is switching to poisson instead of basic func?

Reply:
@AlexLewandowski The `poisson_init` is only used for the ego mission route (which is usually south-west). The poisson distribution gives a nice inflow of social vehicles in the mission route: "nice" in terms it does not clog up the south lanes